packages/data/addon/services/bard-dimensions.ts

```diff
@@ -156,7 +163,7 @@ export default class BardDimensionService extends Service {
    * @param {Number} [options.perPage]
    * @returns {BardDimensionArray} - array of bard dimension model objects
    */
-  find(dimension: string, andQueries: TODO, options: TODO) {
+  find(dimension: string, andQueries: TODO, options: TODO = {}) {
```

Comment:
Should it be `andQueries: DimensionFilter[]`?

Reply:
It is the old one, with the `field` property. We updated the type to now exclude that field. This service will be marked deprecated.